src/support/slack/commands/update-sites-audits.js

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import BaseCommand from './base.js';
+import { extractURLFromSlackInput } from '../../../utils/slack/base.js';
+import { ConfigurationDto } from '../../../dto/configuration.js';
+
+const PHRASES = ['audits'];
+const ERROR_MESSAGE_PREFIX = ':nuclear-warning: ';
+
+export default (context) => {
+  const baseCommand = BaseCommand({
+    id: 'sites--audits',
+    name: 'Update Sites Audits Configuration',
+    description: 'Enables or disables audit functionality for multiple sites.',
+    phrases: PHRASES,
+    usageText: `${PHRASES[0]} {enable/disable} {site1,site2,...} {auditType1,auditType2,...}`,
```

Comment:
we do not currently have the requirement to enable/disable multiple audits for multiple sites at the same time. In fact, this could create some problems with the current data model (ie overriding each other). It would be much simpler for now to have a simple command as following:

```
@spacecat audit enable site.com broken-backlinks
```

cc'ing @alinarublea and @solaris007 for the data model concerns regarding multi update

Reply:
For simplicity, you can enable/disable just one audit type, instead of multiple types simultaniously

Reply:
Hi @ekremney and @alinarublea,

Thank you for your feedback. I’ve updated the Slack command to handle only single-site/single-audit requests.

Reply:
After discussing with @ekremney and according to the https://github.com/adobe/spacecat-api-service/issues/115, I think this would be optimal for the API consumers
```
POST /configurations/sites/audits/{auditType}
{ enabled: true, baseURLs: ['https://site1.com', 'https://site2.com'] } 
```